### PR TITLE
Bugfix/game jam

### PR DESCRIPTION
**Pull request by m4rtint**
Mostly controller and map fixes

### CODE REVIEW

**Comment by m4rtint** [comment]
[Game Jam bugs](https://trello.com/c/4XMKAYpU/99-game-jam-bugs)

File: Assets/InControl/Source/VersionInfo.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public static VersionInfo InControlVersion()
 				Major = 1,
 				Minor = 4,
 				Patch = 4,
-				Build = 3956
+				Build = 3978
```
Comment (mikeslgoh) [inline Assets/InControl/Source/VersionInfo.cs]: So what is this version info for?

File: Assets/Prefabs/Parking/ParkingEight Variant.prefab [hunk 2]
```diff
@@ -870,7 +870,7 @@ PrefabInstance:
     - target: {fileID: 4548340580577678184, guid: 0bb5a62f3a099c147942d55a80f0a329,
         type: 3}
       propertyPath: m_RootOrder
-      value: 7
+      value: 3
```
Comment (mikeslgoh) [inline Assets/Prefabs/Parking/ParkingEight Variant.prefab]: So did you find what was wrong with the parking spot?

File: Assets/InControl/Source/VersionInfo.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/InControl/Source/VersionInfo.cs]: InControl does this thing, idk why. The file is needed locally. Technically can gitignore

File: Assets/Prefabs/Parking/ParkingEight Variant.prefab (hunk 2 above)
Comment (m4rtint) [reply Assets/Prefabs/Parking/ParkingEight Variant.prefab]: I haven't found the issue yet, i'm just reorganizing the spot for now.

File: Assets/InControl/Source/VersionInfo.cs (hunk 1 above)
Comment (mikeslgoh) [reply Assets/InControl/Source/VersionInfo.cs]: Yeah maybe we should add to gitIgnore coz it keeps climbing but no real reason for it besides InControl needing it.

**Comment by m4rtint** [comment]
@mikeslgoh 
For the Controller. I removed the InControl GameObject from the map scene
dontDestroyOnLoad -> True on Main Menu
Removed InControl object from all the game maps scenes.

File: Assets/InControl/Source/VersionInfo.cs (hunk 1 above)
Comment (m4rtint) [reply Assets/InControl/Source/VersionInfo.cs]: realized u can't gitignore it. 
If u clone it, u'lll need this file.
but possible to change this file so it doesn't keep changing

**Comment by m4rtint** [comment]
[Photoshop/Edit car indicator arrow](https://trello.com/c/fZ4cMwIp/88-photoshop-edit-car-indicator-arrow)

**Comment by m4rtint** [comment]
[Photoshop/Edit car indicator arrow](https://trello.com/c/fZ4cMwIp/88-photoshop-edit-car-indicator-arrow)

File: Assets/InControl/Source/VersionInfo.cs (hunk 1 above)
Comment (mikeslgoh) [reply Assets/InControl/Source/VersionInfo.cs]: hmm if it's quite a bit of work to go in and change it then leaving it is fine.

**Comment by Gursaanj** [comment]
Could you add the Canvas with the Pause and End Game panel on the NPC Scenes, then all should be fine. Alternatively I could do that too

**Comment by m4rtint** [comment]
@Gursaanj 
u can add changes to this branch with the pause/endgame panel on npc scenes if u want.if u haven't done it, i will do it

**Comment by m4rtint** [comment]
@Gursaanj 
I placed the user interface panels onto the AI Scenes.

**Pull request merged**